README.md

```diff
@@ -56,58 +56,73 @@ We currently do not:
 
 [Docker](https://www.docker.com/) is the prefered development environment.  Ensure you have Docker Engine installed or otherwise can run containers.
 
-First, open `ci/.docker-env.example`, fill in values as desired, and save as `ci/.docker-env`.  Then build the "notification_api" Docker image by running this command:
+`.local.env` contains all necessary environmental variables and is read automatically in local development and github actions that do not deploy to AWS.
 
-```bash
-docker-compose -f ci/docker-compose-local.yml build app
-```
 
-**Rebuild notification_api whenever Dockerfile.local or requirements-app.txt changes.**
+**Rebuild `notification_api` whenever Dockerfile or poetry.lock changes.**
 
-The associated container will have your local notification-api/ directory mounted in read-only mode, and Flask will run in development mode.  Changes you make to the code should trigger Flask to restart on the container.
+The associated container will have your local notification-api/ directory mounted in read-write mode, and Flask will run in development mode.  Changes you make to the code should trigger Flask to restart on the container. The volume can be commented out in `ci/docker-compose.local.yml`, but you will have to also comment out the `ENTRYPOINT` because `scripts/save_certificate.sh` saves data to the filesystem.
 
 ### Run the local Docker containers
 
 To run the app, and its ecosystem, locally, run:
 
 ```bash
-docker-compose -f ci/docker-compose-local.yml up
+docker-compose -f ci/docker-compose-local.yml build app && docker-compose -f ci/docker-compose-local.yml up
```

Comment:
It is not more concise, it changes the operation. It builds **all**, but that is not what we want. We need to build the app first because multiple services rely on the `notification_api` image. It's always been that way, unfortunately. 

I'd suggest you try and run it as you expressed and then look over the docker-compose-local.yml to help your understanding grow. Should only take 10 minutes. If you want to chat about it afterwords I'd be happy to.